SBTWO-12443 Sep -> Sept

WarwickGoatee opened the pull request:
Added a thing to one of the test HTMLs and it seems to work okay.

halfninja commented:
May need an addition to the README to explain that there's a breaking change here and the latest version may not work with server platforms that use "Sep", such as Java version before 17.

halfninja commented:
This change to [CLDR](https://cldr.unicode.org/downloads/cldr-38) only affects en_gb and en_au - others did not change and should stay as Sep

WarwickGoatee commented:
I've reduced it down to just GB, AU and the default js file.

I've also updated the version to 2.3.0 to mark the breaking change and added a note to the README.
